packages/infra/lib/ihe-stack.ts

```diff
@@ -39,92 +42,128 @@ export class IHEStack extends Stack {
       domainName: props.config.host,
     });
 
+    // get the certificate from ACM
+    const certificate = cert.Certificate.fromCertificateArn(
+      this,
+      "IHECertificate",
+      props.config.iheGateway.certArn
+    );
+
+    // get the ownership Certificate from ACM.
+    const ownershipCertificate = cert.Certificate.fromCertificateArn(
+      this,
+      "OwnershipVerificationCertificate",
+      props.config.iheGateway.ownershipVerificationCertArn
+    );
+
+    // get the truststore bucket from S3
+    const bucket = s3.Bucket.fromBucketName(
+      this,
+      "TruststoreBucket",
+      props.config.iheGateway?.trustStoreBucketName
+    );
```

Comment:
where is this bucket being created?

if we created this bucket manually, we'll need to fix it - we should only create resources manually on AWS only if absolutely necessary

Reply:
Issue here is I tried to create it via cdk, but then deployment fails since the trust bundle isnt inside the bucket. Chicken and egg issue but I think it justifies leaving the bucket manually created for now